contracts/AnchorRepository.sol

```diff
@@ -3,15 +3,14 @@ pragma solidity ^0.4.24;
 import "zos-lib/contracts/Initializable.sol";
 import "openzeppelin-eth/contracts/cryptography/MerkleProof.sol";
 import "contracts/Identity.sol";
-import "contracts/IdentityRegistry.sol";
 
 
 contract AnchorRepository is Initializable {
 
   event AnchorCommitted(
     address indexed from,
     uint256 indexed anchorId,
-    uint48 indexed centrifugeId,
+    address indexed centrifugeId,
```

Comment:
Maybe we could rename the `centrifugeId` to `identityAddress`? What do you think? It might be easier to understand.

Reply:
from a naming perspective I think centrifugeId makes more sense because. The fact that it's an address of a contract is a technicality that should not be reflected in the naming. Maybe we change it back in the future :)

Reply:
@lucasvo

Reply:
BTW, that parameter will be gone soon as we can use msg.sender, so escalating it seems like a waste of time

Reply:
Yea, but from a naming perspective, just in case you need it as a param, I think centrifugeId is a coined term in our org and we should use that as I said earlier. It used across different areas and it more user friendly than identityAddress that is very eth specific. Using the same generic name across applications makes it easier to follow and you don't need to do refactoring. It will suck if every codebase will have a different name for such a central thing like the Centrifuge Id, wouldn't you agree?